core/src/main/scala/org/apache/spark/util/collection/OpenHashSet.scala

```diff
@@ -278,7 +279,7 @@ object OpenHashSet {
 
   val INVALID_POS = -1
   val NONEXISTENCE_MASK = 0x80000000
-  val POSITION_MASK = 0xEFFFFFF
+  val POSITION_MASK = 0x7FFFFFFF
```

Comment:
As these are masks, I'm not sure that's more readable. A hex string is what I would expect.